Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM cusspvz/node:5.1.1
```

Comment:
> could you add reasons you think your node docker image is better than the official https://hub.docker.com/_/node/ ?

I don't think it is better than the official one. But it is more versatile though.
It has the same abilities the official and others provide, but also has a development image targeted for the CD cycle.

cusspvz/node.docker@0e77a04 - I'm currently doing some job on the repo so I can add other *targets*, so I had to bypass faster some issues I was experiencing on *travis*. The `--no-check-certificate` change will **OBVIOUSLY** be rollbacked.

> there seems already many implementations, could you document why you think your own implementation [3] is better than others

It is documented, please read the `README` and the ones that exist per feature.
Also, try to install native bindings on others, I also got it covered because I had issues with all the others. :p